helm-charts/stable/prow-control-plane/templates/config-ConfigMap.yaml

```diff
@@ -94,12 +94,15 @@ data:
         - do-not-merge/hold
         - do-not-merge/work-in-progress
         - do-not-merge/invalid-owners-file
-        repos:
-        {{-  range $repo := .Values.repositories }}
-          - {{ $repo.org }}/{{ $repo.name }}
-        {{- end }}
+        repos: 
+          {{- if eq (len .Values.repositories) 0 }} []
+          {{- else }}
+          {{-  range $repo := .Values.repositories }}
+        - {{ $repo.org }}/{{ $repo.name }}
+          {{- end }}
       merge_method:
-      {{-  range $repo := .Values.repositories }}
-        {{ $repo.org }}/{{ $repo.name }}: squash
-      {{- end }}
+          {{-  range $repo := .Values.repositories }}
+          {{ $repo.org }}/{{ $repo.name }}: squash
+          {{- end }}
+          {{- end }}
```

Comment:
What does this `end` correspond to?

Reply:
The `end` on line 106 corresponds to `range` on line 104
The `end` on line 107 corresponds to `else` on line 99

Reply:
oh great, can you adjust the indentation on the outermost if/else/end for clarity?

Reply:
Yes, I will do that